xbmc/cores/RetroPlayer/rendering/VideoRenderers/RPBaseRenderer.cpp

```diff
@@ -129,8 +134,10 @@ void CRPBaseRenderer::ManageRenderArea(const IRenderBuffer& renderBuffer)
   const unsigned int sourceRotationDegCCW = renderBuffer.GetRotation();
   const float sourceAspectRatio =
       static_cast<float>(sourceWidth) / static_cast<float>(sourceHeight);
-
+  const SCALINGMETHOD scaleMode = m_renderSettings.VideoSettings().GetScalingMethod();
   const STRETCHMODE stretchMode = m_renderSettings.VideoSettings().GetRenderStretchMode();
+  const bool bhw_filter =
```

Comment:
drop the `b` from `bhw` as we don't need to prefix with types

Reply:
Sure, I will do it.